dockstore-client/src/main/java/io/dockstore/client/cli/nested/CromwellLauncher.java

```diff
@@ -83,18 +83,14 @@ public void initialize() {
     @Override
     public List<String> buildRunCommand() {
         final List<String> runCommand;
-        // Don't use imports option for WDL, only for CWL
-        if (zippedEntry == null || abstractEntryClient instanceof ToolClient || Objects.equals(languageType, DescriptorLanguage.WDL)) {
-            runCommand = Lists.newArrayList(primaryDescriptor.getAbsolutePath(), "--inputs", provisionedParameterFile.getAbsolutePath());
-        } else {
-            runCommand = Lists.newArrayList(primaryDescriptor.getAbsolutePath(), "--inputs", provisionedParameterFile.getAbsolutePath(), "--imports", zippedEntry
-                    .getAbsolutePath());
-        }
-
+        runCommand = Lists.newArrayList(primaryDescriptor.getAbsolutePath(), "--inputs", provisionedParameterFile.getAbsolutePath());
         final String[] s = { "java", "-jar", executionFile.getAbsolutePath(), "run" };
         List<String> arguments = new ArrayList<>();
         arguments.addAll(Arrays.asList(s));
         arguments.addAll(runCommand);
+        if (languageType == DescriptorLanguage.CWL) {
+            arguments.addAll(CWL_SPECIFIC_FLAGS);
```

Comment:
Nitpick: it seems like this is only used here, so keep it as local scope https://refactoring.com/catalog/reduceScopeOfVariable.html